tests/test_models.py

```diff
@@ -394,19 +420,45 @@ def test_node_update_fn(self):
         global_attributes = None
 
         node_update_fn = hk.transform(node_update_fn)
-        node_update_fn = hk.without_apply_rng(node_update_fn)
         params = node_update_fn.init(
             init_rng,
             nodes, sent_message, received_message, global_attributes
         )
 
         nodes_updated = node_update_fn.apply(
-            params,
+            params, rng,
             nodes, sent_message, received_message, global_attributes
         )
         np.testing.assert_allclose(
             nodes_updated,
             shifted_softplus(received_message['messages']) + nodes)
 
+    def test_readout_function(self):
+        """Test the output of the readout function against the expected result.
+        """
+        rng = jax.random.PRNGKey(42)
+        rng, init_rng = jax.random.split(rng)
+
+        latent_size = 16
+        hk_init = hk.initializers.Identity()
+        readout_node_update_fn = get_readout_node_update_fn(
+            latent_size, hk_init, use_layer_norm=False,
+            activation=shifted_softplus, output_size=1)
+        # TODO: finish test
+
+    def test_mlp(self):
```

Comment:
can you rename this test to test_build_mlp. Makes it easier to track down the method's associated test